.github/workflows/failed-workflow.yml

```diff
@@ -0,0 +1,45 @@
+##
+# Performs follow-up tasks when a workflow fails or is cancelled.
+##
+name: Failed Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: 'ID of the GitHub Action workflow run to rerun'
+        required: true
+        type: 'string'
+
+jobs:
+  # Attempts to rerun a workflow.
+  #
+  # Performs the following steps:
+  # - Retrieves the workflow run that dispatched this workflow.
+  # - Restarts all failed jobs when the workflow fails or is cancelled for the first time.
+  failed-workflow:
+    name: Rerun a workflow
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - name: Rerun a workflow
+        uses: desrosj/github-script@update/plugin-rest-endpoint-methods # v6.1.1
```

Comment:
Looks like https://github.com/actions/github-script/pull/283 being merged will make this no longer necesary

Reply:
Merged and a new release has been made. Update the PR to point to actions/github-script@v6.2.0.